composer.json

```diff
@@ -16,22 +16,22 @@
 		}
 	],
 	"require": {
-		"php": ">=5.3.2",
+		"php": ">=5.6",
 		"composer/installers": "*",
-		"silverstripe/framework": "^3.1",
-		"guzzle/guzzle": "^3.9",
+		"silverstripe/framework": "4.0.x-dev",
```

Comment:
We should use `^4` since it's stable now.

Worth adding `"silverstripe/assets": "^1"` too since `File` is referenced in the code

Reply:
Done.